tests/Unit/Questions/Question/IsInLineTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace AnthonyEdmonds\GovukLaravel\Tests\Unit\Questions\Question;
+
+use AnthonyEdmonds\GovukLaravel\Questions\Question;
+use AnthonyEdmonds\GovukLaravel\Tests\TestCase;
+
+class IsInLineTest extends TestCase
+{
+    protected Question $question;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->question = new Question('Duck', 'Dave', Question::CHECKBOXES);
+        $this->question->isInline();
+    }
+
+    public function testIsInLineTest(): void
+    {
+        $this->assertTrue(true, "Yes");
```

Comment:
This test is incorrect, you are testing whether true is true, which it always will be!